exampleCourse/questions/template/multiple-choice/images/question.html

```diff
@@ -0,0 +1,19 @@
+<pl-question-panel>
+  <p>Which photo was not taken in the Champaign-Urbana area?</p>
+</pl-question-panel>
+
+<pl-multiple-choice answers-name="ans">
+  <pl-answer correct="true">
+    <pl-figure file-name={{params.correct_filename}} directory="clientFilesQuestion"></pl-figure>
+  </pl-answer>
+  <pl-answer correct="false">
+    <pl-figure file-name={{params.wrong_filename0}} directory="clientFilesQuestion"></pl-figure>
+  </pl-answer>
+  <pl-answer correct="false">
+    <pl-figure file-name={{params.wrong_filename1}} directory="clientFilesQuestion"></pl-figure>
+  </pl-answer>
+  <pl-answer correct="false">
+    <pl-figure file-name={{params.wrong_filename2}} directory="clientFilesQuestion"></pl-figure>
+  </pl-answer>
```

Comment:
Is there a good reason to do things this way with `params` as opposed to listing all correct/incorrect answer and using `number-answers` to pick a subset?

Reply:
You are correct -- we should let `pl-multiple-choice` do this for us.

Reply:
I removed `server.py` entirely in 9479702.